src/callbacks_step/analysis.jl

```diff
@@ -219,7 +219,7 @@ function (analysis_callback::AnalysisCallback)(integrator)
 
     # Calculate current time derivative (needed for semidiscrete entropy time derivative, residual, etc.)
     du_ode = first(get_tmp_cache(integrator))
-    @notimeit timer() rhs!(du_ode, integrator.u, semi, t)
+    @notimeit timer() integrator.f(du_ode, integrator.u, semi, t)
```

Comment:
```suggestion
    @notimeit timer() integrator.f(du_ode, integrator.u, semi, t) # `integrator.f` is usually just a call to `rhs!`
```
Maybe a comment like this to make it easier to understand what's happening here?

Reply:
I expanded the comment a bit.